src/main/java/org/candlepin/service/exception/CouldNotAcquireCloudAccountLockException.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (c) 2009 - 2023 Red Hat, Inc.
+ *
+ * This software is licensed to you under the GNU General Public License,
+ * version 2 (GPLv2). There is NO WARRANTY for this software, express or
+ * implied, including the implied warranties of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv2
+ * along with this software; if not, see
+ * http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+ *
+ * Red Hat trademarks are not licensed under GPLv2. No permission is
+ * granted to use or replicate Red Hat trademarks that are incorporated
+ * in this software or its documentation.
+ */
+
+package org.candlepin.service.exception;
+
+
+
+/**
+ * The CouldNotAcquireCloudAccountLockException is used when organization already being created and/or
+ * entitled
+ */
+public class CouldNotAcquireCloudAccountLockException extends RuntimeException {
```

Comment:
Just a thought. Keeping these exceptions unchecked will give us more options on how to handle them, unless we are ok with mixing checked/unchecked withing a single interface. We won't be forced to handle them by the compiler, but I would argue we are not handling them anyway. I wrote some thoughts on this in comments on Will's PR.

In most of the cases we are just wrapping and rethrowing them. We could introduce a parent exception to make general handling easier. This would let us handle it in places like jobs, but in other places, we could just let them go without polluting the method declarations and leave handling to some centralized handler like what we have for CandlepinException (CandlepinExceptionMapper).

Reply:
Let's discuss this next week when Will and Stepan are back

Reply:
change by discution at java meeting